.eslintrc.js

```diff
@@ -29,6 +30,7 @@ module.exports = {
       "error",
       "always"
     ],
-    "no-console": 0
+    "no-console": 0,
+    "no-mixed-spaces-and-tabs": ["error", "smart-tabs"]
```

Comment:
Typically entire org has the same linting rules
For node-sinek it is:
https://github.com/nodefluent/node-sinek/blob/master/.eslintrc.js

I would preffer no smart-tabs tbh. 
Generally I see linting as separate issue/PR so we can get typescript there with still history or original changes and then relint the code based on rules that can be applied to every project - not just kafka-streams.

Linting will also obfuscate review and will make it hard to spot some issues with migration.